04-connection_and_https/https/setting_up_https.md

```diff
@@ -167,12 +170,16 @@ http {
 
         # Tell the browser that we always want the user to connect via
         # HTTPS to this site.
-        add_header Strict-Transport-Security "max-age=31536000";
+        add_header Strict-Transport-Security "max-age=31536000; includeSubdomains; preload";
 
         # Enable OCSP stapling
         ssl_stapling on;
         resolver 8.8.8.8 8.8.4.4 valid=300s;  # DNS servers to reference
         resolver_timeout 5s;
+        
+        # Diffie-Hellman for forward secrecy
+        # It can be generated using `openssl dhparam -out dhparam.pem 4096`
+        ssl_dhparam /etc/ssl/certs/dhparam.pem;
```

Comment:
Well since the user could have installed nginx in a different path, either intentionally or because it's system doesn't put nginx in `/etc`, and because different permissions might apply requiring `sudo`, I find that we could let it as an exercise to the user to generate it in it's own proper folder and change the path accordingly if need to.

I have added a statement to express this though. If it's not fine to you I can change it again.